packages/governance/src/electorateTools.js

```diff
@@ -42,11 +42,11 @@ const startCounter = async (
 
 /** @param {Store<Handle<'Question'>, QuestionRecord>} questionStore */
 const getOpenQuestions = async questionStore => {
-  const isOpenPQuestions = questionStore
-    .entries()
-    .map(([key, { publicFacet }]) => {
+  const isOpenPQuestions = [...questionStore.entries()].map(
```

Comment:
asking to learn:   does this change  the semantics, or it a safer way of invoking the same operation?  When is it important to map over a copy?

Reply:
(Replaying verbal explanation for the record): This is a consequence of `store.entries()` no longer returning an array, but instead returning only an iterable. The surrounding `[... stuff]` syntax is the easiest way to turn an iterable back into the array. Sometimes the better answer would be not handle it as an iterable without flattening. However, the easiest correctness-restoring change is just to flatten.